README.md

```diff
@@ -92,43 +92,133 @@ Requirements:
 
 For enhanced usability, we provide a [GNU make][1] compatible [Makefile](Makefile).
 
+The nginx-datadog module **depends on two git submodules**:
+  - [libddwaf](https://github.com/DataDog/libddwaf)
+  - [dd-trace-cpp](https://github.com/DataDog/dd-trace-cpp)
+
+Building the module with any of the below commands will init and update them. Otherwise you can do it yourself with:
+```bash
+git submodule update --init --recursive
+```
+
+The easiest way to build the project is through [Docker][2].
+
+Build module using Docker
+---------------
+### Linux, MacOS AMD64
 ```shell
-NGINX_VERSION=1.25.2 make build
+NGINX_VERSION=1.25.2 make build-musl
```

Comment:
I suspect this is incorrect for macOS. Can you check if this command really builds a macOS-compatible binary?

Reply:
If i launch this command:
NGINX_VERSION=1.25.2 ARCH=x86_64 make build-and-test (which will use build_musl). It will work for the build version (Mac ARM can simulate amd64). Notes that it would not be necessary to specify the arch on mac AMD64. However, during tests, one is failing and one crashes (which is RUM and that is weird because I did not enable it)

Reply:
You are right, it does not build a macOS-compatible binary